### ISSUE TITLE
User interface of a Home Page Reader

### ISSUE THREAD

**Issue by Wotipati** [Wotipati/Blind_Accessibility_Papers#197]
### Links
- Paper: https://dl.acm.org/doi/abs/10.1145/274497.274526

<img width="409" alt="ScreenShot 2020-05-17 13 52 16" src="https://user-images.githubusercontent.com/26996041/82136173-baa8a000-9845-11ea-92be-680990b7a573.png">

### Abstract
We first discuss the difficulties that blind people face in trying to lie in society, because of the lack of accessible information resources, and then consider the potential of the Web as a new information resource for the blind. After describing how blind people in Japan currently access the Web, we give an overview of our system for nonvisual Web access. Our system has five special characteristics. One is the use of a numeric keypad for surfing the Net, with a key assignment designed for intuitive operation. The second is a fast- forward key for quick reading. The next two are that hyperlinks are read in a female voice and HTML tags are converted into voice data. The fifth is that the system can be synchronized with Netscape Navigator. After evaluating the system and offering some conclusions, we discuss our plans for future work.  

- ホームページリーダを提案した論文
